lark/reconstruct.py

```diff
@@ -50,18 +39,15 @@ def __default__(self, data, children, meta):
             else:
                 x = next(iter_args)
                 if isinstance(x, list):
-                    to_write += x
+                    to_write.extend(x)
```

Comment:
This changes shows that you do not know how python works. `+=` for lists is exactly equivalent to `.extend` (probably faster since it's not a name lookup), see documentation.